doc/fluid.rst

```diff
@@ -0,0 +1,226 @@
+$$
+
+with gas density $\rho$, species diffusivities ${d}_{\alpha}$, and
+species mass fractions ${Y}_{\alpha}$.  The parens $(\alpha)$ indicate no sum
+over repeated indices is to be performed.
+
+
+.. _heat-flux:
+
+Heat flux
+---------
+
+The total heat flux $\b{q}$ is calculated as the sum of the
+conductive and diffusive components, $\b{q}_{c}$ and $\b{q}_{d}$,
+respectively:
+
+$$
+\b{q} = \b{q}_c + \b{q}_d
+$$
+
+Conductive heat flux
+^^^^^^^^^^^^^^^^^^^^
+The conductive heat flux vector is defined directly from Fourier's law of thermal
+conduction:
+
+$$
+\b{q}_c = -\kappa\nabla{T},
+$$
+
+where $\kappa$ is the thermal conductivity, and ${T}$ is the gas
+temperature.
+
+Diffusive heat flux
+^^^^^^^^^^^^^^^^^^^
+The diffusive heat flux vector is defined as
+
+$$
+\b{q}_d = {h}_{\alpha}\b{J}_{\alpha},
+$$
+
+with the species specific enthalpy ${h}_{\alpha}$, and the species
+diffusive flux vector $\b{J}_{\alpha}$.
+
+.. _chemistry:
+
+Chemistry
+---------
+
+Chemical reactions introduce source terms in the energy and species conservation equations.
+The species source term is the amount of mass produced for each species:
+
+$$
+W^{\mathtt{chem}}_{\alpha} = w_{(\alpha)}\partial_t{\omega}_{\alpha},
+$$
+
+where ${w}_{\alpha}$ is the molecular weight of each species, and
+$\partial_t{\omega}_{\alpha}$ is the net chemical production rate for each species. Here,
+the parens $(\alpha)$ indicates no sum is to be performed over repeated indices.
+
+The energy source term is the amount of thermal energy used to create each species:
+
+$$
+E^{\mathtt{chem}} = -h^f_{\alpha}W^{\mathtt{chem}}_{\alpha},
+$$
+
+where $h^f_{\alpha}$ is the enthalpy of formation for each species.
+
+.. _eos-and-matprop:
+
+Equations of State and Material properties
+------------------------------------------
+
+Equations of state (EOS) provide functions that relate the fluid state $\b{Q}$,
+and the thermodynamic properties such as pressure $p$, temperature $T$, specific
+enthalpies $h_{\alpha}$, and total energy $E$.  The EOS provided by |mirgecom| are
+documented in :mod:`mirgecom.eos`.
+
+Material properties including the first coefficient of viscosity, $\mu$, bulk viscosity
+$\mu_B$, thermal conductivity $\kappa$, and species diffusivities ${d}_{\alpha}$ depend on
+the state of the fluid $\b{Q}$, in general, and are provided by transport models.
+Transport models provided by |mirgecom| will be documented in the forthcoming
+transport module.
+
+.. note::
+
+  The EOS and transport models provide closure for the fluid model in that the fluid
+  thermal state variables such as pressure $p$, temperature $T$, and material
+  properties such as viscosity $\mu$, and thermal conductivity $\kappa$ are functions of
+  the current fluid state $\b{Q}$. The EOS and transport models provide constructs
+  that manage the relationships between these quantities, and provide methods for
+  calculating them from minimal working sets of input data.
```

Comment:
4-wide indentation? (like in all Python-y things?)

Reply:
(f754f60)